swarm/network/stream/messages.go

```diff
@@ -269,9 +269,6 @@ func (m TakeoverProofMsg) String() string {
 
 func (p *Peer) handleTakeoverProofMsg(req *TakeoverProofMsg) error {
 	_, err := p.getServer(req.Stream)
-	if err != nil {
```

Comment:
This if is left because the function is not implemented, to avoid a mistake not checking it then the functionality is implemented. It is fine to remove the check, just to be careful in the future development. This was probably a linter warning.

Reply:
I realized that, but the linter complained, and if we want a green build, we have to adhere to it.

Reply:
👍